### PR TITLE
Fix running multiple scripts on Windows

### PR DESCRIPTION
**Pull request by xPaw**
There's basically no other way besides making a standalone script or writing JS code for it.

### CODE REVIEW

**Comment by YaManicKill** [comment]
:+1: It's not a great way, but I'd much rather this than a custom script.

File: package.json [hunk 1]
```diff
@@ -12,12 +12,12 @@
   },
   "scripts": {
     "start": "node index",
-    "build": "npm run build:grunt && npm run build:handlebars",
+    "build": "(npm run build:grunt || true) && (npm run build:handlebars || true)",
```
Comment (astorije) [inline package.json]: I get the need for `test`, but I think we should let `build` fail if any of these fail. I can't think of a use case for this.

Comment (YaManicKill) [inline package.json]: Actually, that's a good point.

**Comment by astorije** [comment]
This is gross :(
I didn't think we were ensuring compatibility with `cmd`, but if this is really a requirement, then :+1:.

> but I'd much rather this than a custom script.

Yes!

**Comment by astorije** [comment]
Well :+1: with a minor comment inline actually.

**Pull request merged**